openapi/openapi.yaml

```diff
@@ -246,4 +245,64 @@ paths:
         '200':
           description: The current User
         'default':
-          description: Error
+          $ref: '#/components/responses/Error'
+
+components:
+  schemas:
+    Error:
+      title: 'Error'
+      description: "A JSON API Error object"
+      type: object
+      externalDocs:
+        description: 'JSON API Error Specification'
+        url: 'https://jsonapi.org/format/#errors'
+      properties:
+        errors:
+          type: array
+          title: 'ErrorItems'
+          items:
+            $ref: '#/components/schemas/ErrorItem'
+          minItems: 1
+      required:
+        - errors
+
+    ErrorItem:
+      title: 'ErrorItem'
+      description: 'An item returned in the Error "errors" value'
+      externalDocs:
+        description: 'JSON API Error Specification'
+        url: 'https://jsonapi.org/format/#errors'
+      type: object
+      properties:
```

Comment:
JSON API Error format defines also `title` parameter, which differs from `detail`. It seems that in provided examples
https://jsonapi.org/examples/ a `detail` parameter extends the `title`. Shouldn't we include `title` as well and use it as a required one instead of `detail`?

Reply:
The set of JSON API Error attributes used here are based on what the fields in galaxy v2 errors.
ie,
 'code' is v2's 'code
 'detail' is drfs APIException.detail/default_detail  ('message' in the response dict)
'status' is string'ified APIException.status_code
'source.parameter' is 'field'

There isn't a good parallel to 'source.pointer' in current galaxy so could be removed here.

But that's all based on old API, if we want to include 'title' and 'detail' in the errors for the new API I'll add that. 

Would it make sense to go ahead and model the full JSON API Error object here? I was trying to keep it to fields
we currently use, but sense they are all option in JSON API we could add them.

Reply:
'title' added.